back-end/src/controllers/verifyController.js

```diff
@@ -0,0 +1,40 @@
+const client = require('twilio')(process.env.ACCOUNT_SID, process.env.AUTH_TOKEN);
+ 
+ 
+exports.getCode = async (req, res) => {
+    console.log("******** getCode");
+    console.log(req.query.phonenumber);
+    console.log("channel: "+req.query.channel);
+    client
+        .verify
+        .services(process.env.VERIFY_SERVICE_SID)
+        .verifications
+        .create({
+            to: `+${req.query.phonenumber}`,
+            channel: req.query.channel
+        })
+        .then(data => {
+            console.log("res of getCode:"),
+            console.log(data),
+            res.status(200).send(data)
+        })
+};
```

Comment:
This has a security flaw. The back-end should check the request came from Slowvid. Otherwise a malicious user could use `curl` and send many sms to any phone number.

It should be rate limited to mitigate against an attack. Something like: rate limit of 3 requests per Slowvid front-end per day.